src/components/dialogs/parameters/dynamicsimulation/curve/country-select.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useMemo } from 'react';
+import CheckboxSelect from './common/checkbox-select';
+
+const CountrySelect = ({ value, options, onChange }) => {
+    // take from CountrySelector
```

Comment:
make comment more explicit ? why can't we share more with utils/localized-countries-hook.js ?

Reply:
OK, will be refactorized in another PR...